examples/load-data-via-graph-construction.ipynb

```diff
@@ -47,20 +44,19 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "id": "izJD_HlcAdQ0"
```

Comment:
the id is something that annoyed me recently as well. I think @nvitucci  said it can be useful if you want to reference a specifc cell though

Reply:
The cell IDs have been introduced in [nbformat 4.5](https://nbformat.readthedocs.io/en/latest/format_description.html#cell-ids). We could get rid of them, but this might create issues in the future; it did create an error in the conversion process that I cannot find again ATM.

I'm wondering if it would make sense to just have them added at once to all notebooks and update the `nbformat` metadata.